src/h5p-editor-csv-to-text.ts

```diff
@@ -38,6 +39,11 @@ class CSVToTextWidget extends H5PWidget<H5PFieldText> implements IH5PWidget {
       },
     );
 
+    const label = H5PEditor.createLabel(field);
+    this.wrapper.innerHTML += label;
+    this.wrapper.classList.add("field");
+    this.wrapper.classList.add("field-name-textField");
```

Comment:
I _think_ `field-name-X` is a dynamic class generated with `field.name`. Please verify before commiting.

```suggestion
    this.wrapper.classList.add(`field-name-${field.name}`);
```

Reply:
True! Added this now